src/js/queries/topServicesQuery.js

```diff
@@ -0,0 +1,19 @@
+const topServicesQuery = `
+  query topServices {
+    allServicePages(first: 4) {
```

Comment:
just in line with the naming comment below -- I think allServicePages is sorta confusing.

Reply:
I don't think its as simple as renaming without changing the joplin code. 
```
{
  "errors": [
    {
      "locations": [
        {
          "column": 5,
          "line": 2
        }
      ],
      "message": "Cannot query field \"topServices\" on type \"Query\". Did you mean \"allServicePages\"?"
    }
  ]
}
```
It makes ok sense to me that the topServices would be a subset (the first 4) of all service pages. I think it might make sense to have a topServices query in joplin in the future, but maybe it would be best as an editable within the CMS featuredServices? idk.

Reply:
@benweatherman, any thoughts about what this query might be?